docs/DEVELOPER.md

```diff
@@ -0,0 +1,499 @@
+## Hey developer!
+
+Would you like to improve, or add new components to `mynah-ui`? Then here's a guide for you, which will both tell the basics, rules and a complete guide to a functional component additions.
+
+### Before you start!
+
+__There are a few key things you need to keep in mind all the time.__
+
+- As you can see from the project type on github _(which is a guess by github)_, it might look like `mynah-ui` is an `SCSS` project. And to be fair, it is not completely wrong. We're trying to solve as much as we can with css. On the typescript/code side, we're just generating dom elements, making calculations and more importantly: toggling css classes on and off for the elements we create. 
```

Comment:
> We're trying to solve as much as we can with css

Very useful insight. Helps other devs understand the strategy/positioning of this library.

Reply:
Thanks🙏
In later, we're also trying to inform the developers that if we see any kind of inline styles etc, it will be rejected ;)